mlir/test/Dialect/Rock/regularize.mlir

```diff
@@ -0,0 +1,44 @@
+// RUN: rocmlir-opt --rock-regularize %s | FileCheck %s
```

Comment:
Thanks for adding this!

Reply:
I agree with unit testing as well, and yes it is lacking. Just a bit frustrated with the e2e resnet kernel tests that also checked for a rigid IR, and most of these were very similar so they didn't add much new coverage.